From 64f4ffd33c5d37f4bce368db44b9fd12742b4916 Mon Sep 17 00:00:00 2001
From: Roshan Pius <rpius@google.com>
Date: Mon, 5 Feb 2018 16:16:57 -0800
Subject: [PATCH 06/10] klte-common: wpa_supplicant(hidl): Add support for
 starting HAL lazily

Bug: 72394251
Test: Able to start supplicant from framework using
ISupplicant.getService()
Change-Id: I19b8434e7241b9028e7dc86316ec9d5512affcca
---
 rootdir/etc/init.qcom.rc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rootdir/etc/init.qcom.rc b/rootdir/etc/init.qcom.rc
index 7250889..7f508b7 100644
--- a/rootdir/etc/init.qcom.rc
+++ b/rootdir/etc/init.qcom.rc
@@ -919,6 +919,8 @@ service qmuxd /vendor/bin/qmuxd
 service wpa_supplicant /vendor/bin/hw/wpa_supplicant \
     -O/data/vendor/wifi/wpa/sockets -puse_p2p_group_interface=1 \
     -g@android:wpa_wlan0
+    interface android.hardware.wifi.supplicant@1.0::ISupplicant default
+    interface android.hardware.wifi.supplicant@1.1::ISupplicant default
     class main
     socket wpa_wlan0 dgram 660 wifi wifi
     disabled
-- 
2.17.1

